scene/gui/line_edit.cpp

```diff
@@ -1669,6 +1679,8 @@ LineEdit::LineEdit() {
 	pass = false;
 	secret_character = "*";
 	text_changed_dirty = false;
+	placeholder = "";
+	placeholder_translated = "";
```

Comment:
Strings are default initialized to `""` (so `String() == ""`), so this is not necessary.